internal/examples/supervisor/bin/supervisor.yaml

```
@@ -0,0 +1,5 @@
+server:
+  endpoint: ws://127.0.0.1:4320/v1/opamp
+
+agent:
+  executable: ../../../../../collector-contrib/bin/otelcontribcol_darwin_amd64
```

```suggestion
  executable: ../../../../../opentelemetry-collector-contrib/bin/otelcontribcol_darwin_amd64
```

It is not significant, but if someone has opamp-go and the collector cloned using the default project names, then this will make it easier to run it locally :)

---

Fixed, thanks.